### DISCUSSION TITLE
Tips：通过开启字幕功能，可以将音频也纳入记录检索 | By turning on the subtitles function, audio can also be included in the record search

### DISCUSSION THREAD

**Discussion by Antonoko** [Antonoko/Windrecorder#65] [Show and tell 显摆酷玩意]
在 Windows 11 22H2 之后，通过开启系统自带的字幕功能，即可在看无字幕视频、直播、会议、线上聊天等场景下，将音频内容转录为显示于屏幕上的字幕文字，从而纳入搜索的检索范围。当使用飞书会议等支持实时字幕转录的会议软件时，通过打开字幕区域可实现对线上会议的记录备份。

After Windows 11 22H2, by turning on the system's built-in subtitle function, you can transcribe the audio content into subtitle text displayed on the screen when watching unsubtitled videos, live broadcasts, meetings, online chats, etc., and then include it in searches search scope. When using conferencing software that supports real-time subtitle transcription, such as Feishu Meeting, you can back up the records of the online meeting by opening the subtitle area.

![image](https://github.com/yuka-friends/Windrecorder/assets/11193477/f5e9b721-1401-4093-b628-46b6424144d8)

**Comment by ASC8384** [comment]
https://trac.ffmpeg.org/wiki/Capture/Desktop#UseDirectShow

FFmpeg 也支持声音的录制。

**Reply by Antonoko** [reply]
确实，也许可以作为一个可选项（虽然个人不用，我的记录需求只需大概的画面，现在这个记录帧率配声音的体验大概不会太好（
